tests/Aspire.Dashboard.Tests/Integration/Playwright/AppBarTests.cs

```diff
@@ -104,7 +104,8 @@ await Assertions
 
                 await AsyncTestHelpers.AssertIsTrueRetryAsync(
                     async () => await checkbox.IsCheckedAsync(),
-                    "Checkbox isn't immediately checked.");
+                    "Checkbox isn't immediately checked.",
+                    retries: 15 /* this seems to take a very long time to run, so needs a long timeout */);
```

Comment:
Does 5 extra retries increase the timeout a lot?

Reply:
Yes, it does. It increases total timeout by ~40 seconds